### PR TITLE
Societal Grant 2

### PR DESCRIPTION
**Pull request by gfox1**
#### Project Abstract

This is the second grant for the Societal Labs team, after the sucessful submission and merge of our first W3F grant. 

- Previous Grant: [Societal.md](https://github.com/w3f/Grants-Program/pull/1249) 
- Milestone 1 [Submission](https://github.com/w3f/Grant-Milestone-Delivery/pull/628)
- Milestone 2 [Submission](https://github.com/w3f/Grant-Milestone-Delivery/pull/665)

Societal is a specialized blockchain for the creation and management of Decentralize Autonomous Organizations (DAOs). Societal allows all types of groups or communities to build their own online, transparent, and decentralized organization. Societal bundles all of the tools required to create and manage a DAO in one place. Creators will be empowered to construct a DAO with fungible, non-fungible, or a combination of governance tokens. 

Societal also offers DAO management tooling features like treasury management, specialized governance, task boards, legal structuring, and accounting. This removes the need to utilize siloed platforms to manage the operations of a DAO. Whether a creator is looking to build a DAO for their organization, raise and deploy investment capital, or decentralize governance of an NFT project, Societal has the necessary tooling for a seamless end-to-end experience.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] A BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (aUSD/USDT) address for the payment of the milestones is provided in the application.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the new grant application. I have one initial question: The UI mockups that you shared, are these the ones that will be delivered as part of the grant? Apart from this, my main personal concern is that the application at the moment is relatively expansive, especially given that you want to release your own token. Ideally, this work is also funded by VC or investors that actually take a look at your business model. We mostly just look at these things from a technical perspective.

**Comment by gfox1** [comment]
Hi @Noc2, thanks for getting to this so quickly! 

In regards to the UI mockups, these show the final version of the application with all of its features. All of these pages will not be delivered in the grant but an in depth Governance page will be delivered. This governance page will have the same governance features as Governance V1 like - Vote Delegation, Adaptive-Quorum Biasing and Conviction Voting. We are working with our designer to decide on how to best display these features and bring forward the advanced governance features that Gavin Wood has created to all DAOs in the DOT ecosystem and not just the native chains. The governance page would be in the same branding / design guidelines as you see there. 

To interact with these governance features however, we need to add the Lockable and Reservable Traits to the current assets pallet. This is what is outlined in milestone 1 and we believe that it would be a great benefit to the ecosystem if the assets pallet had these features. With the Web 3 Foundation's review of this milestone, we believe that our code delivery could be merged directly into the current assets pallet in the  official substrate repo, and be a new version of it. Thus directly giving back to everyone that uses the assets pallet going forward. 

I do understand your concern about funding items from a technical perspective and not for business use cases. If we have to update our grant application to reflect only the technical side than this is alright. We thought however, that is would be best to deliver the technical items in milestone one and then show a use case / how to use them with milestone 2.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the quick reply. I will mark your application as ready for review so that others also take a look at it. As I said above, it would be nice if you could reduce the price. For example, a level 2 grant for 30k would also be easier to approve, but that is just my opinion.

**Comment by keeganquigley** [comment]
Thanks for the interesting application @gfox1.

- In my experience, the MetaMask wallet is not widely used in the Polkadot/Kusama ecosystem (outside of Moonbeam/Moonriver). Will you consider connecting the front-end to other popular wallets such as Talisman, Fearless, Polkadot{JS}, etc.?
- M1 seems a bit expensive for adding traits to a pallet.
- M2 especially seems expensive for creating the UI. Would you consider reducing the price?

Thanks!

**Comment by gfox1** [comment]
Hi @Noc2 and @keeganquigley, thanks for your comments!

Based on your suggestions I have lowered the price of the grant to a level 2 grant for 30k. Please see this reflected in the latest commit. 

@keeganquigley, in regards to the other wallet connections, so far we have integrated Polkadot.js, Talisman and MetaMask. Please see the image below. We are also looking to integrate SubWallet in the coming days.  We do intend all features to be used with a Polkadot based wallet. The reason for the MetaMask integration is to get DAOs from Ethereum to start using our chain and hence a Polkadot backend for their governance 😄 

image

**Comment by keeganquigley** [comment]
@gfox1 thanks for the changes! Much appreciated. Just a few follow-up questions:

- You mentioned in your original application that you planned to integrate aUSD as your main stable coin. Just curious if this is this still the case?
- Your gas free model is a first of its kind for an EVM based substrate chain as far as I'm aware. Can you give us an update on your progress in this area? Have you done any testing against DDoS attacks, etc.? How does this work alongside [previously mentioned](https://github.com/w3f/Grants-Program/pull/1450/files#diff-14ecafc4356eb801054beb90a22429bda218a6171d8d4964b01951268c6be5f3R52) cross-chain smart contract executions, since they are more computationally expensive?

File: applications/societal_grant2.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+- https://github.com/asansyzb
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/graemefox/
+- https://www.linkedin.com/in/tngell/
+
+## Development Status :open_book:
+
+Societal has already been working on our project and has completed our MVP. The two main repositories of the Societal project are the [societal-node](https://github.com/sctllabs/societal-node) and the [societal-client](https://github.com/sctllabs/societal-client). These repesortiries represent the societal blockchain (built with substrate) and the UI. To date or MVP includes the following features: Create a DAO with its own governance token, elect council members, create proposals, and vote on them with your MetaMask wallet.
```
Comment (keeganquigley) [inline applications/societal_grant2.md:104]:
```suggestion
Societal has already been working on our project and has completed our MVP. The two main repositories of the Societal project are the [societal-node](https://github.com/sctllabs/societal-node) and the [societal-client](https://github.com/sctllabs/societal-client). These repositories represent the societal blockchain (built with substrate) and the UI. To date or MVP includes the following features: Create a DAO with its own governance token, elect council members, create proposals, and vote on them with your MetaMask wallet.
```

**Comment by gfox1** [comment]
Hi @keeganquigley, no problem at all. Please find below answers to your follow-up questions:


> * You mentioned in your original application that you planned to integrate aUSD as your main stable coin. Just curious if this is this still the case?

- As of right now, USDT will be the first stable coin that we will integrate to our platform. Our original application came before USDT was availble on Polkadot. However, we would still like to integrate aUSD in the future. 

> * Your gas free model is a first of its kind for an EVM based substrate chain as far as I'm aware. Can you give us an update on your progress in this area? Have you done any testing against DDoS attacks, etc.? How does this work alongside [previously mentioned](https://github.com/w3f/Grants-Program/pull/1450/files#diff-14ecafc4356eb801054beb90a22429bda218a6171d8d4964b01951268c6be5f3R52) cross-chain smart contract executions, since they are more computationally expensive?

- We have not made any progress on this to date, as we have not had the resources to tackle this problem. After we have a working application for the creation and management of DAOs, we will then begin to tackle this feature. 
- However, from a high level we will charge the DAOs a monthly subscription which will eliminate the gas fees for the members of the DAO. We believe that this will greatly increase DAO participation and will also be necessary if we want Ethereum based DAOs to use our platform. To prevent DDoS attacks, DAOs will have a maximum number of transactions their members can make on-chain / cross-chain per month. Members will be subjected to a max number of proposals, vote per proposal, etc., so they don't spam the chain. We have listed our subscription models [here](https://docs.sctl.xyz/societal-platform/pricing), but in enhance this feature is to increase member participation.  We do not want DAO members to have any in participating in DAOs, paying for participation in the DAO and will charge the DAOs treasury for the transactions its members make. 

Hope this answers your questions!

**Review by keeganquigley — APPROVED** [review]
@gfox1 cool! Thanks for the details. You touched exactly where I was going with that. Having previously been a member of a couple DAOs, I know that often voter turnout is pretty low even _without_ having to pay. So it's good to know that essentially the DAO can subscribe and pay the funds out of the treasury rather than member's pockets. 

Though I think the gas-free concept would appeal to many, I haven't heard of any other chains implementing it so far, and since substrate weights are already pretty cheap (Statemint/Statemine fees are even smaller), it might be better to stick with traditional tx models if it will jeopardize the users in any way.

That being said, I definitely think we need more DAO tools and some sleek UIs for DAO members who are used to off-chain voting on platforms like Discord, so I am happy to approve it and look forward to using it!

**Review by Noc2 — APPROVED** [review]
Thanks for the update. I'm happy to go ahead with it.

**Comment by gfox1** [comment]
Hey @keeganquigley, thanks for the suggestions. We definitely want to do what is best for the DAO members and if the gas free isn't working we are open to pivoting. Looking forward to you using it as well and thanks for approving the grant! 

@Noc2 and @randombishop, thanks as well for approving the grant!

**Pull request merged**